lua/nvim-tree/keymap-legacy.lua

```diff
@@ -401,7 +402,7 @@ function M.cmd_generate_on_attach()
     return
   end
 
-  local name = "/tmp/my_on_attach.lua"
+  local name = utils.path_join { vim.fn.stdpath "cache", "my_on_attach.lua" }
```

Comment:
Retrospectively could have named that file `nvim-tree-on-attach.lua` or something like that. :)

Reply:
Done. Updating wiki after merge.